cube/cube.gd

```diff
@@ -52,6 +52,7 @@ func _on_mouse_exited():
 
 func reveal_cube():
 	if !is_cleared and !is_flagged:
+		$RevealCube.play()
```

Comment:
It might be smart to define relations to other nodes at the top of the file, so there's only one place to change when refactoring (moving / renaming nodes)